libs/components/forms/src/lib/modules/checkbox/checkbox.component.ts

```diff
@@ -66,12 +66,6 @@ export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
     this.inputId = `input-${this.#_id}`;
   }
 
-  // TODO: In a future breaking change - remove this getter. It is not used - only the `inputId` is used;
-  // however, this component is returned by the `SkyCheckboxChange` and thus removing this would be a breaking change.
-  public get id(): string {
```

Comment:
Are we doing the ID one in this PR too? I thought we were doing separate PRs?

Reply:
I needed to remove this for code coverage purposes. The only way we were calling this getter was through a test that inspected it on the source property we're removing.